whitepaper/cacts_arch.svg

```diff
@@ -0,0 +1,1777 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
@sfuji822 This looks like it has the same content as `./whitepaper/cactus_arch.svg`, do we really need this or could it be deleted to avoid dupliation?

Reply:
I will delete it.